WhaleSpotting/react-app/src/styles/Navbar.scss

```
@@ -0,0 +1,68 @@
+@import './constants.scss';
```

this needs to be capitalised
```suggestion
@import './Constants.scss';
```